index.html

```diff
@@ -1,54 +1,277 @@
+  <header>
+    <div class="flex-container">
+      <div class="logo">
+        <h2>
+          <a href="#">My Logo</a>
+        </h2>
+        <img src="/assets/images/Icon-Menu.jpg" alt="Icon menu" />
+      </div>
+      <div class="header-wrapper">
+        <div class="text">
+          <h2>I’m Mariane <br />Glad to see you!</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+        </div>
+        <div class="social-media">
+          <div class="icon-a">
+            <h5><a href="#">Let's connect</a></h5>
+          </div>
+          <ul>
+            <li>
+              <a href="#">
+                <img src="assets/images/twitter.svg" alt="twitter image" class="twitter" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="assets/images/linkedin.svg" alt="linked in  image" class="linkedin" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="assets/images/m.svg" alt="meduim in  image" class="meduim" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="assets/images/github.svg" alt="github in  image" class="github" />
+              </a>
 
-        </p>
+            </li>
+            <li>
+              <a href="#">
+                <img src="assets/images/angleList.svg" alt="angle list in  image" class="anglelist" />
+              </a>
+            </li>
+          </ul>
+
+        </div>
+      </div>
     </div>
-    <div class="stories-buttons">
+  </header>
+  <div class="wrapper">
+    <section class="work-wrapper">
+      <div class="work-section">
+        <div class="snapshoot-potofolio"></div>
+        <div class="work-txt">
+          <h2>Tonic</h2>
+          <h5 class="first-element-ul">CANOPY</h5>
+          <ul class="list-project">
+            <li><img src="/assets/images/Counter.png" alt="counter" /></li>
+            <li>Back End Dev</li>
+            <li><img src="/assets/images/Counter.png" alt="counter" /></li>
+            <li>2015</li>
+          </ul>
+          <div class="work-paragraph">
+            <p>
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </p>
+          </div>
+          <div class="work-buttons">
+            <ul>
+              <li class="html"><a href="#">html</a></li>
+              <li class="css"><a href="#">css</a></li>
+              <li class="js"><a href="#">javascript</a></li>
+            </ul>
+          </div>
+          <div class="see-project-btn">
+            <a href="#">See Projects</a>
+          </div>
```

Comment:
- [ **OPTIONAL** ] Kindly make sure all your buttons follow the button styling: `Active`, `Hover` and `Pressed` (see design) I suggest you apply that styling for all `See project`, and `Get My Resume` buttons to make your project match to the Figma template.
You can pick the used colors from Figma on this image below :point_down: :point_down: 
![buttons](https://user-images.githubusercontent.com/83042742/171971620-fcc115cb-cdea-4a41-b573-6f6dafdc81e0.png)